usaspending_api/awards/v2/filters/idv_filters.py

```diff
@@ -38,17 +41,17 @@ def idv_transaction_filter(filters):
     return TransactionNormalized.objects.filter(award_id=idv_award_id)
 
 
-def idv_treasury_account_funding_filter(account_type, download_table, filters, account_level):
+def idv_federal_account_funding_filter(account_type, download_table, filters, account_level):
     if account_level != "treasury_account":
```

Comment:
Renamed function to federal account, but the next line will raise an Exception if it is for federal account

Reply:
Yeah.  I feel like there's no winning with the function name.  It's running against the tas table, but it's called the `idv_federal_account_funding` download.  Probably should have just left it alone.  Oh well.  Ross is in a hurry to get this onto QAT so they can test more easily so I'm just going to leave it.  Sorry future developers for messing everything up!

Reply:
Actually, I'll make a follow-up pull request and rename this back.  I found a comment that I forgot to update.  /sigh

"The hurrier I go, the behinder I get."